packages/react/src/themes/base/components/Dialog/dialogStyles.ts

```diff
@@ -12,21 +12,37 @@ export default {
     padding: v.rootPadding,
     position: 'relative',
     width: v.rootWidth,
-    display: 'flex',
-    flexWrap: 'wrap',
+    // CSS Grid is polifilled only with latest inline-style-prefixer
+    // @ts-ignore is supported by fallback values plugin
+    display: ['grid', '-ms-grid'],
+    gridTemplateColumns: '1fr auto',
+    msGridColumns: `1fr auto`,
```

Comment:
We will not need to do this manually after #1743. Can you please use stringified properties instead? It will be easier to remove them.

```suggestion
    'ms-grid-columns`: '1fr auto',
```